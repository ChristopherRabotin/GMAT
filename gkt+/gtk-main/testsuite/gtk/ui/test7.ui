<interface>
  <object class="GtkBox">
    <object class="GtkBox">
    </object>
  </object>
</interface>
