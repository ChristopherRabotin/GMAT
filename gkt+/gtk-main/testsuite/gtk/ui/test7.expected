ERROR: gtk-builder-error-quark 4
<input>:4:1 <object> is not a valid tag here
